streamer/src/nonblocking/quic.rs

```diff
@@ -628,6 +633,7 @@ async fn packet_batch_sender(
     trace!("enter packet_batch_sender");
     let mut batch_start_time = Instant::now();
     loop {
+        let mut packet_perf_measure: HashMap<[u8; 64], std::time::Instant> = HashMap::default();
```

Comment:
if we have 64 packets, and 1/4096 chance of this being a traced-packet, we can pre-allocate this map with some small size.
Also could we just use a Vec to avoid hashing?

Seems like most often we will have 0 traced packets, and if there are any there's probably just 1-2.

Reply:
Will look!

Reply:
Done